scripts/setup-testsuite-dataset

```diff
@@ -82,6 +81,11 @@ if [ -z "${HONEYCOMB_DATASET+x}" ]; then
   exit 255
 fi
 
+if [ -z "${HONEYCOMB_API_ENDPOINT+x}" ]; then
```

Comment:
I think setting this here is too late as `COLUMNS_API` and `DATASETS_API` will already be set with the empty string?

Reply:
from the "test us" output:

```
HONEYCOMB_API_ENDPOINT is not set, using default https://api.honeycomb.io/
2024-04-18T14:27:20+0000 error creating dataset "testacc"
2024-04-18T14:27:20+0000 created dataset "testacc}"
2024-04-18T14:27:20+0000 error creating column "trace.trace_id"
2024-04-18T14:27:20+0000 created column "trace.trace_id" in "testacc"
2024-04-18T14:27:20+0000 error creating column "app.tenant"
2024-04-18T14:27:20+0000 created column "app.tenant" in "testacc"
2024-04-18T14:27:20+0000 error creating column "trace.span_id"
2024-04-18T14:27:20+0000 created column "trace.span_id" in "testacc"
2024-04-18T14:27:20+0000 error creating column "duration_ms"
2024-04-18T14:27:20+0000 created column "duration_ms" in "testacc"
2024-04-18T14:27:20+0000 error creating column "trace.parent_id"
2024-04-18T14:27:20+0000 created column "trace.parent_id" in "testacc"
2024-04-18T14:27:20+0000 error creating column "app.error"
2024-04-18T14:27:20+0000 created column "app.error" in "testacc"
2024-04-18T14:27:20+0000 error creating column "service_name"
2024-04-18T14:27:20+0000 created column "service_name" in "testacc"
2024-04-18T14:27:20+0000 error creating column "column_2"
2024-04-18T14:27:20+0000 created column "column_2" in "testacc"
2024-04-18T14:27:20+0000 error creating column "column_1"
2024-04-18T14:27:20+0000 created column "column_1" in "testacc"
2024-04-18T14:27:20+0000 error creating column "name"
2024-04-18T14:27:20+0000 created column "name" in "testacc"
```

Which.. also means that script failing isn't failing the run, I guess :|

Reply:
🙃 that's what I get for assuming green = good and not poking further

Reply:
It _should_ have caused "red = bad" 🙃 

We can fix that outside of this, tho'